book/chapters/09-boot/contents.tex

```diff
@@ -535,48 +557,49 @@ \section{Bootloader}
 versiuni diferite de imagini de kernel (fișiere ce conțin sistemul de operare)
 ale aceluiași sistem de operare, fișiere aflate pe aceeași partiție.
 
-În cazul folosirii BIOS, încărcarea bootloaderului se face în doi pași:
+În cazul folosirii BIOS, încărcarea bootloader-ului se face în doi pași:
 
 \begin{enumerate}
   \item întâi se încarcă prima fază din bootloader (numită \textit{1st stage bootloader}), o secvență scurtă care se găsește în MBR (\textit{Master
     Boot Record})
   \item ulterior, \textit{1st stage bootloader} încarcă a doua fază din bootloader
-    (\textit{2nd stage bootloader}), o secvență mai detaliată care se găsește
-		într-un fișier pe disc și care deține codul pentru a putea opta
-		între diferite sisteme de operare și pentru a configura
-		opțiunile de boot pentru kernel
+    (\textit{2nd stage bootloader}), o secvență mai detaliată care se poate afla
+    într-un fișier pe disc, in VBR(Volume boot record) sau în spațiul dintre MBR și prima partiție.
+    Și care deține codul pentru a putea opta
+    între diferite sisteme de operare și pentru a configura
+    opțiunile de boot pentru kernel
 \end{enumerate}
 
 În cazul UEFI, partiția ESP (\textit{EFI System Partition}) conține fișiere executabile
-\texttt{.efi} care vor încărca sistemul de operare. Aceste fișiere executabile sunt
-echivalentul bootloaderului în cazul BIOS. Fiind o tehnologie main nouă, UEFI
-nu folosește un bootloader în mai multe faze ci încarcă direct din partiția ESP
-fișierul necesar.
+\texttt{.efi} acestea pot contine functionalitate arbitrara, dar cel mai des sunt un bootloader
```

Comment:
. Aceste fișiere pot conține orice funcționalitate, dar cel mai adesea conțin codul unui boorloader sau chiar nucleul sistemului de operare ce trebuie bootat.